src/main/java/edu/kis/vh/nursery/FIFORyhmer.java

```diff
@@ -16,6 +16,12 @@ public FIFORyhmer(IntStack intStack) {
 		super(intStack);
 	}
 
+	/**
+	 * Bez bicia przyznam się, że nie wiem co to zasada izolacji, ale obstawiłbym, że właśnie w tej metodzie
+	 * jest ona złamana. Używany jest tu jednocześnie zbiór elementów z nadklasy jak i tymczasowy zbiór temp
```

Comment:
Insulation is briefly explained in the beginning of lab instruction. Breaking insulation in bridge pattern comes from the explicit reference to IntLinkedList type.